Src/AutoFixture/EmailAddressLocalPart.cs

```diff
@@ -0,0 +1,142 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace Ploeh.AutoFixture
+{
+    /// <summary>
+    /// Represents the local part of the email address, defined as everything up to, but not including the @ sign.  
+    /// The requirements enforced on the local part are defined in RFC 3696.  The regular expression for validating 
+    /// the local part borrowed from Phil Haack at 
+    /// http://haacked.com/archive/2007/08/21/i-knew-how-to-validate-an-email-address-until-i.aspx/
```

Comment:
Perhaps I was a bit too fast on the trigger there :flushed: 

I think I've misremembered the [Framework Design Guidelines on this topic](https://msdn.microsoft.com/en-us/library/ms229027.aspx), but now that I read them again, they seem to be aligned with the CA rule.

So, despite the coupling, it may be most appropriate to keep them unnested. Sorry about the inconvenience.